observations/README.md

```diff
@@ -0,0 +1,1279 @@
+            <translation value="130.000">
+               <originalText>mm[Hg]</originalText>
+            </translation>
+         </value>
+      </ObservationStatement>
+   </component>
+   <Participant typeCode="PRF" contextControlCode="OP">
+      <agentRef classCode="AGNT">
+         <id root="6D6BF46C-476B-4955-AE07-CB53C1D9CC40"/>
+      </agentRef>
+   </Participant>
+</CompoundStatement>
+```
+</details>
+
+## Comment Note (JSON FHIR > XML HL7)
+
+Where a FHIR `Observation` resource is coded as a Comment Note (37331000000100) and it is not part of an investigation,
+it is mapped to a HL7 `NarrativeStatement` as follows.
+
+| Mapped to (XML HL7 NarrativeStatement) | Mapped from (JSON FHIR / other source )                                                                   |
+|----------------------------------------|-----------------------------------------------------------------------------------------------------------|
+| id                                     | Unique ID generated by the adaptor                                                                        |
+| text                                   | `Observation.comment`                                                                                     |
+| statusCode                             | fixed value = `"COMPLETE"`                                                                                |
+| availabilityTime \[@value]             | `Observation.effectiveDateTime` or else `Observation.effectivePeriod.start` or else `Observation.issued`  |
+| Participant \[@typecode]               | fixed value = `"PRF"`                                                                                     |
+| Participant / agentRef / id \[@root]   | `Observation.performer[0]`                                                                                | 
+
+
+
+<details>
+   <summary>Example XML</summary>
+
+```XML
+
+<NarrativeStatement classCode="OBS" moodCode="EVN">
+   <id root="D2FD6804-2CB9-48D7-950E-E616661C887C"/>
+   <text>This is a free text note under history</text>
+   <statusCode code="COMPLETE"/>
+   <availabilityTime value="20100206124100"/>
+   <Participant typeCode="PRF" contextControlCode="OP">
+      <agentRef classCode="AGNT">
+         <id root="6D6BF46C-476B-4955-AE07-CB53C1D9CC40"/>
+      </agentRef>
+   </Participant>
+</NarrativeStatement>
+```
+</details>
+
+## Investigations
+
+The Test Group / Result structure is itself nested as a component of the Specimen `compoundStatement` (not shown here), 
+which forms part if the [Diagnostic Report](../diagnostic%20reports/README.md) structure.
+
+### Narrative statement mapping for Test Group Header and Test Result
+
+Where stated in the Test Group Header and Test Result mapping, a collection of `NarrativeStatement` components may be 
+used to preserve data. Where this takes place, they will be mapped from the appropriate `ObservationStatement` in the following way:
```

Comment:
> mapped from the appropriate `ObservationStatement` in the following way

I'm confused by this, as I thought `ObservationStatement` was an HL7 concept, and we're talking about FHIR -> HL7 here. 🤔

Reply:
Changed to `Observation`, thanks